src/webportal/src/plugins/job-submission/components/CompletionSection.jsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) Microsoft Corporation
+ * All rights reserved.
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import React from 'react';
+import {Stack} from 'office-ui-fabric-react';
+import PropTypes from 'prop-types';
+import {BasicSection} from './BasicSection';
+import {Completion} from '../models/completion';
+import {CSpinButton} from './CustomizedComponents';
+
+export const CompletionSection= (props) => {
+  const {onChange, value} = props;
+  const {minFailedInstances, minSuceedInstances} = value;
+
+  const _onChange = (keyName, newValue) => {
+    if (onChange === undefined) {
+      return;
+    }
+    const completion = new Completion(value);
+    completion[keyName] = newValue;
+    onChange(completion);
+  };
+
+  return (
+    <BasicSection sectionLabel={'Completion'} sectionOptional>
+      <Stack horizontal gap='s1'>
+        <CSpinButton label={'Min Failed Instances'}
+                    value={minFailedInstances === undefined? NaN.toString() : minFailedInstances}
```

Comment:
minFailedInstances and minSucceededInstances have default values.
https://github.com/microsoft/pai/blob/8fc8c46dc9a6ed6d88efc43cc33c823c13ab0883/docs/pai-job-protocol.yaml#L50

Reply:
I will fix it in the object constructor, thanks for this comments